lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -704,7 +720,12 @@ void _MaterialXData::_FixLibraryTangentInputs(mx::DocumentPtr& mtlxDoc)
         for (mx::NodePtr node : nodeGraph->getNodes()) {
             mx::NodeDefPtr nodeDef = node->getNodeDef();
             if (!nodeDef) {
+#if MX_COMBINED_VERSION >= 13900
+                // Can happen with legacy data. Just continue.
+                continue;
+#else
```

Comment:
The UsdShade networks in USD can still be 1.38. They might have `node:id` that are no longer in the 1.39 library (`ND_swizzle_*`, `ND_normalmap`). So not being able to get a NodeDef for a USD `node:id` must be treated as "business as usual".